From 9f056842fad21dc69a51fd9f01543cd85a6d4047 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sat, 15 Mar 2014 17:48:41 -0400
Subject: [PATCH 3/3] build: Add BLUETOOTH_LIBS to libgnome-shell's libadd for
 ld.gold

For ld.gold, adding BLUETOOTH_LIBS to gnome_shell_LDADD has no effect:
the libgnome-bluetooth-applet must be linked to the binary that actually
uses a function from libgnome-bluetooth-applet, i.e. to libgnome-shell.

https://bugzilla.gnome.org/show_bug.cgi?id=726435
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index f489a56..79838a3 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -320,7 +320,7 @@ libgnome_shell_base_la_LIBADD = $(libgnome_shell_libadd)
 libgnome_shell_base_la_CPPFLAGS = $(gnome_shell_cflags)
 
 libgnome_shell_la_LDFLAGS = $(libgnome_shell_ldflags)
-libgnome_shell_la_LIBADD = $(GNOME_SHELL_LIBS) $(MUTTER_LIBS) libgnome-shell-base.la
+libgnome_shell_la_LIBADD = $(GNOME_SHELL_LIBS) $(MUTTER_LIBS) $(BLUETOOTH_LIBS) libgnome-shell-base.la
 libgnome_shell_la_CPPFLAGS = $(MUTTER_CFLAGS) $(gnome_shell_cflags)
 
 if HAVE_MUTTER_WAYLAND
-- 
1.9.0

